src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt

```diff
@@ -662,6 +666,8 @@ data class SubaccountOrder(
     val reduceOnly: Boolean,
     val cancelReason: String?,
     val resources: SubaccountOrderResources,
+    val subaccountNumber: Int?,
```

Comment:
just want to double check that this won't cause issues due to changing param order? If we are changing order here, I think we need to touchup `SubaccountOrder` in docs as well.

Reply:
not worth it then! good thinking
d417bc77b118fe256edf130bd88679a1e50fa2b2